metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -314,14 +315,15 @@ func NewContainerMetadataEnricher(
 					// which is in the form of <container.runtime>://<container.id>
```

Comment:
Do you think is time to allign here also with kube_state container.id  where container.runtime is not splitted?

Reply:
Hmm, that is another issue right? Good if we can fix them both.

Reply:
That would be a breaking change. For state_container metricset as far as I see, the container.id is the same form.

Reply:
Indeed. So it is time maybe to test here both container.ids and align the

Reply:
What does fixing mean?  I prefer the splitted container.id

Reply:
Check below the kubernetes.container.id (vs container.id)

![Screenshot 2023-02-08 at 2 45 38 PM](https://user-images.githubusercontent.com/1685415/217533569-564dcd55-8805-42ac-b75d-4727647a614b.png)

I also prefer the splitted, so to remove it from `kubernetes.container.id`

Reply:
@gizas @ChrsMark fixes in https://github.com/elastic/beats/pull/34516/commits/cdc87aea23a19a7ba6f43f2582a082c774f9a93b